rhyme.txt

```diff
@@ -0,0 +1 @@
+"The rain in Spain falls mainly in the plain."
```

Comment:
Correct but this file is supposed to be within a sub directory called `cli-diagnostic`